classes/ObjectModel.php

```diff
@@ -1447,6 +1458,28 @@ public function cacheFieldsRequiredDatabase($all = true)
     }
 
     /**
+     * Get required fields list for this model or for all the models
+     *
+     * @param bool $all : whether it should return required fields for this model or all the models
+     *
+     * @return array
+     */
+    public function getCachedFieldsRequiredDatabase($all = false)
+    {
+        $this->cacheFieldsRequiredDatabase($all);
```

Comment:
Why is it called here?

Reply:
Before, it was called in AddressCore::getFieldsRequiredDB(). But with refactoring, the call was moved in parent class.